scripts/build_renewable_profiles.py

```diff
@@ -603,10 +603,18 @@ def create_scaling_factor(
             columns={"x": "lon", "y": "lat", "country": "countries"}
         ).loc[bus_in_hydrobasins, ["lon", "lat", "countries", "shape_id"]]
 
-        resource["plants"]["installed_hydro"] = [
-            True if (bus_id in hydro_ppls.bus.values) else False
-            for bus_id in resource["plants"].index
-        ]
+        if snakemake.params.alternative_clustering == False:
+            resource["plants"]["installed_hydro"] = [
+                True if (bus_id in hydro_ppls.bus.values) else False
+                for bus_id in resource["plants"].index
+            ]
+
+        ### TODO: quickfix. above case and the below case should by unified
+        if snakemake.params.alternative_clustering == True:
+            resource["plants"]["installed_hydro"] = [
+                True if (bus_id in hydro_ppls.region_id.values) else False
+                for bus_id in resource["plants"].shape_id.values
+            ]
```

Comment:
A great catch! On the first glance, it feels like the TODO can be addressed quite easily, using something like:
 
```suggestion
        if snakemake.params.alternative_clustering == True:
            resource["plants"]["installed_hydro"] = [
                True if (bus_id in hydro_ppls.region_id.values) else False
                for bus_id in resource["plants"].shape_id.values
            ]
            else:
                resource["plants"]["installed_hydro"] = [
                   True if (bus_id in hydro_ppls.bus.values) else False
                   for bus_id in resource["plants"].index
               ]
```
@energyLS does it make sense for you?

Reply:
@ekatef thanks for the suggestion, it indeed makes sense and feel free to commit your suggestion. However, I actually though of a even "deeper" unification, in the sense that 
`hydro_ppls.bus.values`and `hydro_ppls.region_id.values` are merged to one id/index and `resource["plants"].shape_id.values` and `resource["plants"].index` are merged to one id/index no matter wether the alternative clustering is true or false. But I think this might be an extra issue?

Reply:
Agree, it would be perfect to use the generalisation you suggest. Do you have any idea on how to implement it quickly? Probably we can use

```python
resource["plants"]["installed_hydro"] = [True if (bus_id in atlite_hydro_ids) else False for bus_id in ppl_hydro_ids] 
  ```
  
While `atlite_hydro_ids` and `ppl_hydro_ids` will be defined depending on `snakemake.params.alternative_clustering` (naming may be improved for sure!). What do you think?

Reply:
Great reviews :D
Katia anticipated my comments.
As mentioned in the past call, I am ok with these types of temporary fixes.
The solid solution would be instead of dropping completely the inconsistency between alternative yes/no:
if an optional shapefile is passed to build_bus_regions and alternative is enabled, the index of that geojson will be used as index of the following shapes.
The source of all these issues are related to this I believe

Reply:
@ekatef and @davide-f thanks for your inputs, I agree the clean solution would be dropping completely the inconsistency between alternative clustering. However I feel like this is a separate PR?